test/distributed/org/apache/cassandra/fuzz/topology/HarryOnAccordTopologyMixupTest.java

```diff
@@ -71,6 +71,13 @@ public HarryOnAccordState(RandomSource rs)
             this.listener = new ListenerHolder(this);
         }
 
+        @Override
+        protected boolean allowRepair()
+        {
+            //TODO (required, operational): repair barriers depend on durability sync, which requires ALL, even if --force is done.
+            return topologyHistory.down().length == 0;
```

Comment:
disable repair when there are downed nodes.  We use `--force` but seeing barriers complaining in the logs about not enough nodes